.gitignore

```diff
@@ -1,4 +1,4 @@
-config/*
+/config
```

Comment:
```suggestion
/config
config/*
```

To keep existing and allow for your use-case we need both right?

Reply:
No?

`/config` <- can be both a file or folder?

Reply:
dark magic